### PR TITLE
Initial draft. Need to add links and pics.

### PR DESCRIPTION
**Pull request by dpnova**
Work in progress here for now.

### CODE REVIEW

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
...stor expectations against freedom for your team to deliver in an environment that isn't soul crushing. I personally am of the view that which of timelines or features you commit to depends on the maturity of the business you are coming from. If this is a greenfields project with little commercial history in the business the ability to pivot is essential. If you have confidence that your team is capable of learning fast and not putting blinkers on as soon as production starts you could opt to commit to timelines that allow you deliver a prototype without being specific about the exact features. In this case investors are investing in your vision and the team executing on the desire outcome, not the specific features you will be building. The contrast to this if you understand very clearly the feature that needs to be built. Perhaps you have an already running business that understands how a blockchain solution takes things to the next level, or perhaps you already have a working prototype you have validated in-market. In this situation you likely want to commit to features, not timelines. This will allow you to test and get the product-market fit right for your particular end users.
```
Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: and the team executing on the desire**d** outcome

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+If you meet resistance to this approach, it should be easy to justify why each of these approaches is desirable. There's no point in committing to features for a product that is still very conceptual, if people invest because they are expecting a specific feature you are likely going to end up disappointing more than one of your investors. They will all see different features in their head when they invest initially because you haven't been specific about them. They key is to commit to when they will have clarity around their belief in your vision and execution capacity. On the other hand there's no point in committing to aggressive timelines if you have committed to specific features. If people have invested in the amazing features you have clearly defined, then they should understand that perfecting them takes time. Even when they are clearly defined you still need to finesse and test with your market, this all takes time.
```
Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: The~~y~~ key is to commit to...

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Two approaches that are open and provide clarity without digging yourself a hole to dragged into later on in the process.
```
Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: to _be_ dragged into... ?

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## About giving up your seekrits!
+
+In terms of product features etc - I tend to take a slightly different view to many others, so caveat emptor on the following ramblings from me.
+
+I don't believe in competitors, there may be other companies that validate my target market, or create disenfranchised customers for me to pick up. They are both great things. Ultimately success that I've seen has been more based on the support of those who believe and connect with the world view of your product. That comes back to some very lean principles; get out of the building, connect, learn, fail, then produce something that resonates with your customers. There's always going to be people creating similar products.
```
Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: get out of the building and connect / learn / fail, then produce

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+I see it like offices, you walk into a business and make a judgement about how you feel about them based on the way they decorate their office. Some you will connect with, others less so. We can't build the product that satisfies everyone, so make your "office" awesome for you and those close to you, that passion will translate into awesome relationships with your customers and a product experience they connect with.
+
+Now the more blockchain related perspective: TSEs that are most successful are the ones that understand that they are creating something that will ultimately be controlled by the community. The products that fit into this view need to be understood by the community because protocols are only useful if anyone can implement them. If you're running a TSE for a product that will never play nice the community then you're most likely either: a) not being ambitious enough or b) probably not an ideal candidate for running a TSE. Hopefully you're just in (a) because that just means thinking bigger!
```
Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: "Token sale events (TSEs)"

First time use in the post, most people don't know what "TSE" stands for

Comment (ejjick) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: will never play nice **with** the community then

**Comment by dpnova** [comment]
OK now i think @ejjick 

I've added you as a reviewer @jokershaw so you can accept whenever you want.

**Comment by dpnova** [comment]
nudge here @jokershaw

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+If you can think about how your solution will benefit the community, even if you operate it with full control in the early days, you're well onto your way to articulating something that investors will love to see in your whitepaper. It's easy to be open about something that is going to be amazing for a whole community of people. Investors will be happy to accept that you will maintain control initially if they can see the bigger picture.
+
+I'm more worried about selling my product once for the first time than I am about someone else trying to do it too. If you bare all (or something close to it) in a TSE you are going to be able to prove you did it first. Everyone else will copy it one day anyway. Also you will be well funded and have a great community ready to go with the product when it's live!
+
```
Comment (jokershaw) [inline _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: I suggest you close with a summary of what we encourage in this space and how the ES platform provides quality standards and guidelines.  Obviously not a hard sell, more along the lines of haring what you are working on and believe in.

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md (hunk 5 above)
Comment (dpnova) [reply _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: We're avoiding ICO as it carries legal grey area BS about securities...

File: _posts/2018-06-04-baring-it-all-in-your-whitepaper.md (hunk 6 above)
Comment (dpnova) [reply _posts/2018-06-04-baring-it-all-in-your-whitepaper.md]: Good idea, thanks

**Pull request merged**